test/k8sT/microscope.go

```diff
@@ -21,8 +21,11 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var _ = Describe("DisabledK8sMicroscope", func() {
+var (
+	microscopeTestName = "K8sValidatedMicroscope"
```

Comment:
Don't add that, is no longer needed due is only used once, and it's easy to see the testName if the name is on describe.

Reply:
done